sphinx/themes/basic/static/searchtools.js

```diff
@@ -97,6 +97,17 @@ const _displayItem = (item, searchTerms, highlightTerms) => {
     fetch(requestUrl)
       .then((responseData) => responseData.text())
       .then((data) => {
+
+        // exclude results that don't contain exact phrases if we are searching for them
+        if (data) {
+          const lowercaseData = data.toLowerCase();
+          for (const exactSearchPhrase of exactSearchPhrases) {
+            if (lowercaseData.search(exactSearchPhrase) === -1) {
+              return;
+            }
+          }
```

Comment:
```suggestion
          const lowercaseData = data.toLowerCase();
          const mismatch = (s) => lowercaseData.search(s) === -1;
          if (exactSearchPhrases.some(mismatch)) return;
```

This can be simplified as follows.